governance/GOVERNANCE.md

```diff
@@ -57,17 +57,21 @@ To improve organizational transparency and collaboration between the CPC and Boa
 
 ## Consensus Seeking Process
 
-The Team follows a [Consensus Seeking][] decision-making model.
-
-When an agenda item has appeared to reach a consensus, the moderator
-will ask "Does anyone object?" as a final call for dissent from the
-consensus.
-
-If an agenda item cannot reach a consensus, a Team member can call for
-the item to be decided by a vote or to table the issue to the next
-meeting. In both cases the decision must be seconded by a majority of the Team
-or else the discussion will continue. Simple majority wins. Only Active
-Members participate in a vote.
+The Team follows a [Lazy Consensus Seeking][] decision-making model.
+
+Lazy consensus is achieved by making a proposal in a forum that will reach all
+members of the group who should have the opportunity to participate in reaching
+consensus and waiting for an appropriate amount of time for anyone to object.
```

Comment:
How is “appropriate” determined?

Reply:
I think we should define a number like X weeks, and if it's too much or too little we can update it

Reply:
I prefer to keep it loose here, and mention something like "after a CPC public meeting".

Reply:
I had thought about it and different times may apply depending on the group/topic which is why I left it out I think specific times can be defined for specific cases, for example approving a new project might have a longer time than something else.

Reply:
I'm totally fine to keep it loose in the actual document, to be clear, which is why i stamped it - i was trying to get an understanding of what others' intentions/expectations here are.

Reply:
+1 to keeping the definition non-specific here. We could indicate that the timing is context-specific, like so:

```suggestion
consensus and waiting for an appropriate amount of time given the nature of the decision for anyone to object.
```